### DISCUSSION TITLE
Got hacked twice by an old build from this site

### DISCUSSION THREAD

**Discussion by anatheme6969** [community/community#62239] [Code Security] [answered]
### Select Topic Area

Show & Tell

### Body

Someone used a program or build off this site to hack me not once but twice now It was called hockeyapp/bit something, i have one of there user names, and an admin can remotely access my pc to see more info. Im new to this and dont know how to stop it

**Comment by pokemon918** [comment]
That's amazing

**Comment by queenofcorgis** [comment, marked as answer]
Hi @anatheme6969  👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
